### DISCUSSION TITLE
Is there somewhere else apps are stored so I don't have to export them manually?

### DISCUSSION THREAD

**Discussion by HideoKuze** [Budibase/budibase#4981] [Help] [answered]
The IP address of my EC2 instance recently changed and I don't think I can access the old Budibase installation (unless I can?) I'll have to initialize a new one, will my old apps still be there? Or is there somewhere I can locate them in the filesystem?

Or better yet can I point an existing installation to a new IP?

**Comment by mjashanks** [comment, marked as answer]
Your old apps should still be there. You should be able to access via its new IP address.

**Reply by HideoKuze** [reply]
I get a small error: `Error response from daemon: Conflict. The container name "/bbworker" is already in use by container` Is it okay to remove or rename it without issue?

**Reply by HideoKuze** [reply]
I renamed it and it's fine, thank you again
